jubakit/wrapper.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin
+from .classifier import Classifier, Config, Dataset
+
+
+class BaseJubatusClassifier(BaseEstimator, ClassifierMixin):
+  """
+  scikit-learn Wrapper for Jubatus Classifiers.
+  """
+  
+  def __init__(self, n_iter=1, shuffle=False, softmax=False, embedded=True, seed=None):
+    """
+    Creates a base class for Jubatus Classifiers"
+    """
+    self.softmax = softmax
+    self.n_iter = n_iter
+    self.shuffle = shuffle
+    self.embedded = embedded
+    self.seed = seed
+    self.fitted_ = False
+    self.clf_ = None
```

Comment:
Okay, I change `clf` to `classifier`
Note: scikit-learn developers use `clf` for classifier object.
https://github.com/scikit-learn/scikit-learn/blob/b6941847d6f224446917ba52b4467cfbf082fcd8/sklearn/ensemble/voting_classifier.py#L203